docs/src/modules/components/AppSearch.js

```diff
@@ -153,28 +154,31 @@ function NewStartScreen() {
   );
 }
 
-function DocSearchHit(props) {
-  const { children, hit } = props;
+const displayTagProductId = {
+  'material-ui': 'Material UI',
+  'joy-ui': 'Joy UI',
+  'base-ui': 'Base UI',
+  x: 'MUI X',
+  system: 'MUI System',
+  toolpad: 'Toolpad',
```

Comment:
Fix the missing toolpad badge when searching in Toolpad

Before:

<img width="761" alt="Screenshot 2023-06-14 at 17 11 43" src="https://github.com/mui/material-ui/assets/3165635/c585fac9-37e9-4ff6-92fa-dc9b61031517">

https://mui.com/toolpad/getting-started/first-app/

After:

<img width="759" alt="Screenshot 2023-06-14 at 17 14 50" src="https://github.com/mui/material-ui/assets/3165635/ac271154-30aa-4d10-a7b3-e36af18b7d66">

cc @Janpot